modP2pImpl/src/org/aion/p2p/impl/comm/NodeMgr.java

```diff
@@ -73,13 +77,18 @@ public String dumpNodeInfo(String selfShortId, boolean completeInfo) {
                         selfShortId));
         sb.append(
                 String.format(
-                        "temp[%3d] inbound[%3d] outbound[%3d] active[%3d]                                         s - seed node, td - total difficulty, # - block number, bv - binary version\n",
+                        " temp[%3d] inbound[%3d] outbound[%3d] active[%3d]                    id - node short id, s - seed node, td - total difficulty, # - block number, bv - binary version\n",
```

Comment:
please add another 8 spaces before the `id` here to make it the same width as the info below